app/api/score/_utils/getAIResponse.ts

```diff
@@ -0,0 +1,20 @@
+export const runtime = "edge";
+// openAI API setup
+import OpenAI from "openai";
+
+const openai = new OpenAI({
+  apiKey: process.env.OPENAI_KEY,
+});
+
+const getAIResponse = async (resumeText: string, jobDescription: string) => {
+  const response = await openai.completions.create({
+    model: "text-davinci-003",
```

Comment:
should we use the newest models? this one's marked as legacy: https://platform.openai.com/docs/models/gpt-3-5
```suggestion
    model: "gpt-3.5-turbo",
```